notes/stories/television/tv-theamericanshortstory.st.txt

```diff
@@ -406,3 +405,52 @@ fear [Bill Kenner admitted to Harold that "he was scared" fighting in the Great
 Christianity [Mrs. Krebs praised the Lord for having brought her son home from the war safe and sound. It is clear from the small town American context that this was the Christian God.]
 coping with a crisis of faith [While Harold's mother seems distressed that Harold cannot pray aloud with her, Harold seems very matter-of-fact about his unbelief.]
 
+
+theamericanshortstory10
+=======================
+
+:: Title
+The Golden Honeymoon
+
+:: Description
+"The Golden Honeymoon" is the ninth installment of The American Short Story
+television anthology series produced by Learning in Focus and Sea Cliff
+Productions for the Public Broadcasting Service. This installment is based on
+the 1922 Ring Lardner short story of the same name.
+
+Synopsis: An elderly couple go to Florida to celebrate their golden wedding
+anniversary.
+
+:: Date
+1980
+
+:: References
+https://www.imdb.com/title/tt0080804/
```

Comment:
Have we decided to let through IMDb references now? I'm on the fence. I realise Wikipedia doesn't have a specific page for this episode but I don't think that per se should be a reason to add IMDb references if otherwise we avoid IMDb references.

Reply:
I've been using them here and there, but to be on the safe side, I've replaced the tass story IMDb reference urls with the Wikipedia url about the series.